pkgs/tools/X11/inputplug/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub
+, lib
+, libbsd
+, pkg-config
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inputplug";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "andrewshadura";
+    repo = pname;
+    rev = version;
+    sha256 = "02pslkjrc20wjc8zr7fck9djbhyahh25xlva7pcabyp4jlh2kf8f";
+  };
+
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
needs a patch comment

Reply:
Please use the cargo.lock directly instead of using a patch.

Reply:
It shouldn't be needed if you use `fetchCrate`, the crates.io version includes a [`Cargo.lock`](https://docs.rs/crate/inputplug/0.4.0/source/Cargo.lock)